kickstart/etc/nginx-posm.conf

```diff
@@ -34,13 +36,18 @@ server {
   location /fp-tiler {
     proxy_pass http://127.0.0.1:{{fp_tiler_port}};
   }
-  location /fp-tasks {
-    proxy_pass http://127.0.0.1:{{fp_tasks_port}};
+  location /fp-tasks/ {
+    proxy_pass http://127.0.0.1:{{fp_tasks_port}}/;
```

Comment:
You want this trailing slash? Won't this make urls of `http://127.0.0.1:8081//fp-tasks/foo`?